### ISSUE TITLE
add contributor attributes

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/ioos-atn-data#14]
Need `contributor_email`, `contributor_name`, `contributor_role`, `contributor_role_vocabulary`

**Comment by iamchrisser** [comment]
I've manually added these fields to the example file I'll send out later this morning, but because this involves changing the way the registration app collects personnel info and shares that with (or gets it from) other metadata services, we won't be able to actually add these to our production netCDF creation processes for some time. From the NCEI and ATN perspectives, do these fields need to be immediately available before we can finalize and implement this archive process?

**Comment by MathewBiddle** [comment]
I see.
https://github.com/ioos/ioos-atn-data/blob/3c36a14f87c54c6ba154c822514a9072fb38181f/data/examples/atn_trajectory_template_ncdump.txt#L280-L283

In some cases, NCEI will use the list of contributors to build an appropriate citation for the archive package. So, while these are *recommended* attributes, they are important if used for building the archive dataset citation.

For example, in the netCDF file the attribute could look like:
`contributor_name: Antonio Baptista, Joe Needoba, Michael Wilkin, Charles Seaton, Sarah Riseman`

Which has a citation:
Baptista, António; Needoba, Joe; Wilkin, Michael; Seaton, Charles; Riseman, Sarah; Mayorga, Emilio; Northwest Association of Networked Ocean Observing Systems (2017). Oceanographic data collected from SATURN-08 by Center for Coastal Margin Observation and Prediction (CMOP) and assembled by Northwest Association of Networked Ocean Observation Systems (NANOOS) in the Columbia River Estuary and North East Pacific Ocean from 2012-09-04 to 2019-06-14 (NCEI Accession 0162622). [indicate subset used]. NOAA National Centers for Environmental Information. Dataset. https://www.ncei.noaa.gov/archive/accession/0162622. Accessed [date].

So, I guess my answer is how does @mmckinzie want to build out the citations at NCEI? If you only care about giving the `creator_name` attribution in the citation, then we are okay. But, I assume ATN wants to give credit where credit is due.

A last ditch option could be to include the attributes, just don't populate them (yet). Then, NCEI can build the citation using those attributes when they have content.

**Comment by iamchrisser** [comment]
I think we're waiting to hear back from MM about what the ATN office wants citations to look like?

**Comment by MathewBiddle** [comment]
ping @mmckinzie

**Comment by MathewBiddle** [comment]
@mmckinzie @iamchrisser 
We had a good conversation about this yesterday, but I'm not sure where we ended. I think there are two different requirements that are hitting at once.

1. NCEI needs a list of people (comma separated last name, first name) to identify in the citation. The solution to that is to populate the `contributor_name` attribute with the list of people. From NCEI's perspective it doesn't matter how that gets done, if it's pulled from specific fields in the reg app or simply a free text field, NCEI just wants a consistent place to pull the appropriate names.
2. When using the attribute `contributor_name` [ACDD **suggests**](https://wiki.esipfed.org/Attribute_Convention_for_Data_Discovery_1-3#contributor_role) the use of the `contributor_role` attribute (which can be free text or from an identified vocabulary).

If it's too complex to add a full list of contributors to make the appropriate citation, I suggest we drop back 10 and punt. By that, I mean use the `creator_name` as the author (which is already done) and make plans to include all contributors in a later version. 

https://github.com/ioos/ioos-atn-data/blob/3c36a14f87c54c6ba154c822514a9072fb38181f/data/examples/atn_trajectory_template_ncdump.txt#L245

**Comment by iamchrisser** [comment]
These are included in the recent commit as placeholders until we improve the reg app and can actually collect this info.

**Comment by MathewBiddle** [comment]
Looks good: https://github.com/ioos/ioos-atn-data/blob/2dcafb8ee6c47b01e4ca4f8d6042ae07652288e4/data/atn_trajectory_template_ncdump_20211006.txt#L333-L336
